task-management/src/main/java/org/exoplatform/task/management/controller/TaskController.java

```diff
@@ -519,7 +519,7 @@ public Response comment(Long taskId, String comment, Long parentCommentId, Secur
   @Resource
   @Ajax
   @MimeType.HTML
-  public Response loadAllComments(Long taskId, SecurityContext securityContext) throws EntityNotFoundException, UnAuthorizedOperationException {
+  public Response loadAllComments(Long taskId, String lang, SecurityContext securityContext) throws EntityNotFoundException, UnAuthorizedOperationException {
```

Comment:
Here you have the current user, so his locale, no need to modify this to put lang in parameter